### ISSUE TITLE
Add support for retaining completed WUs instead of deleting them

### ISSUE THREAD

**Issue by jchodera** [FoldingAtHome/fah-issues#1187]
@bb30994 noted that it would be useful to retain completed WUs under some circumstances while they are being uploaded (instead of immediately deleting them) for debugging purposes. I'll let him chime in with more details.

**Comment by jcoffland** [comment]
Yes, with out overly complicating the client logic.  Perhaps the client could move finished WUs in to a rotating and pruned log directory.  The client already handles old log files and configs this way so it would be pretty easy to add.

**Comment by bb30994** [comment]
The example we've been looking at had an error and it would be nice to have access to the working files as they were immediately after the error.  Perhaps this would be managed by a debug switch which defaults to ON which creates a tarball and attaches that final state so there's an opportunity to retrieve it at the server.   Under some conditions, this may already be happening, but at least not the one we wanted.  

The ability to store them locally beside the client's working files would probably be less valuable.
